Tasks/PipAuthenticateV1/task.json

```diff
@@ -0,0 +1,98 @@
+{
+  "id": "5e3feff0-c5ae-11e8-a7d0-4bd3b8229800",
+  "name": "PipAuthenticate",
+  "friendlyName": "Python pip authenticate",
+  "description": "Authentication task for the pip client used for installing Python distributions",
+  "author": "Microsoft Corporation",
+  "helpUrl": "https://docs.microsoft.com/azure/devops/pipelines/tasks/package/pip-authenticate",
+  "helpMarkDown": "[Learn more about this task](https://pip.pypa.io/en/stable/reference/pip_install/) or [see the pip documentation](https://pip.pypa.io/)",
+  "category": "Package",
+  "version": {
+      "Major": 1,
+      "Minor": 0,
```

Comment:
Please set minor version to the current sprint. I know it probably won't be released for a little while, but it just solves issues in the long run.

Reply:
thanks for pointing. I clearly missed it.